tools/htdocs/components/20_VEPForm.js

```diff
@@ -107,7 +107,7 @@ Ensembl.Panel.VEPForm = Ensembl.Panel.ToolsForm.extend({
         $(this).data('valid', false);
         return;
       }
-      if ($.trim($(this).find('.vep-input').val()) == "" && $(this).find('.ffile').val() == "" ) {
+      if ($.trim($(this).find('.vep-input').val()) == "" && $(this).find('.ffile').val() == "" && $(this).find('.ftext._url ').val() == "" ) {
```

Comment:
Hi @msmanoj  This should work. 

But I have heard that using underscores are like "_url"  denotes a private variable. Here is the class assigned when we create the dom element here https://github.com/Ensembl/public-plugins/blob/release/102/tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm#L198

How about using "url" instead of "_url" just for safer side?

Reply:
@jyothishnt - Yes, that makes sense. I actually didn't notice that it. has `url` as well. I've updated it